From 52338f7beb7973bcd7a3c882751752e9a0192f59 Mon Sep 17 00:00:00 2001
From: Joel Selvaraj <foss@joelselvaraj.com>
Date: Fri, 13 Jun 2025 22:44:17 -0500
Subject: [PATCH 098/211] stalled! arm64: dts: qcom: sdm845-xiaomi-beryllium:
 Add q6voicedai reg

This makes it similar to the other dais. Noticed in sdm660 repo.  But I am not sure if it's a functional requirement. Anyways let's specify it.

Signed-off-by: Joel Selvaraj <foss@joelselvaraj.com>
---
 .../arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
index 9a877e76b908..1ba5d526baa0 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
@@ -872,3 +872,9 @@ &q6cvs {
 &q6mvm {
 	status = "okay";
 };
+
+&q6voicedai {
+	dai@1 {
+		reg = <VOICEMMODE1>;
+	};
+};
-- 
2.55.0

